components/UserList.tsx

```diff
@@ -126,13 +126,13 @@ function UserList({
       />
       <div className="controls">
         {/*
-          Only permit user creation for admin users.
+          Only permit user creation for certain types of users.
           Other users must be created through standard flows.
         */}
-        {type === 'admin' && (
+        {['admin', 'cdp'].find((t) => t === type) && (
           <Button
             className="mr-3"
-            onClick={onCreateAdminUser}
+            onClick={() => onCreateAdminUser(type)}
```

Comment:
Renamed it in https://github.com/ibi-group/otp-admin-ui/pull/64/commits/fbc1304d508832ad56e3c9330709941b1a039845 because that branch made some other changes to the method